canonical/canonical.com
# feat: /data/kafka/managed build

## Done

- Built page as per [doc](https://docs.google.com/document/d/122FxBfJ1tflDGkCusViawtLeRTIr13PVOT4JUzeFIkU/edit?tab=t.0) and [design](https://www.figma.com/design/uJUwbUAGDog4KCMivrMpnl/24.10-Data-Fabric?node-id=1495-8370&node-type=canvas&m=dev)

## QA

- View the site locally in your web browser at: http://0.0.0.0:8002/data/kafka/managed
- Check against doc and design 

## Issue / Card

Fixes https://warthogs.atlassian.net/browse/WD-15050
